### PR TITLE
Docs: Update 'Whitelisted' to 'Allowed' per style guide

### LINKED ISSUE #38888: Change "whitelisted" label setting in data source configuration
(originally reported by Eve Meelan)
Screen Shot 2021-08-25 at 2 06 31 PM

**What happened**:
In the DataSource/Prometheus configuration page, one of the settings is labeled "Whitelisted Cookies"

**What you expected to happen**:
The label should be "Accepted Cookies"

**Anything else we need to know?**:
Documentation may need to be updated.
The affected file can be found here https://github.com/grafana/grafana/blob/main/packages/grafana-ui/src/components/DataSourceSettings/DataSourceHttpSettings.tsx (thanks to Gio for finding it for me).

### PR DESCRIPTION
**Pull request by ashharrison90**
…e frontend now.

Closes  https://github.com/grafana/grafana/issues/38888

**What this PR does / why we need it**:

- updates the docs to use `Accepted` instead of `Whitelisted`

**Which issue(s) this PR fixes**:

Follow on from https://github.com/grafana/grafana/pull/38886

**Special notes for your reviewer**:

### CODE REVIEW

**Review by dprokop — APPROVED** [review]

File: docs/sources/datasources/influxdb/_index.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ These options apply if you are using the InfluxQL query language. If you are usi
 | `Access`  | Server (default) = URL needs to be accessible from the Grafana backend/server, Browser = URL needs to be accessible from the browser.  |
 
 **Note**: Browser access is deprecated and will be removed in a future release.
-`Whitelisted Cookies`| Cookies that will be forwarded to the data source. All other cookies will be deleted.
+`Accepted Cookies`| Cookies that will be forwarded to the data source. All other cookies will be deleted.
```
Comment (osg-grafana) [inline docs/sources/datasources/influxdb/_index.md:38]:
```suggestion
`Allowed cookies`| Cookies that will be forwarded to the data source. All other cookies will be deleted.
```

File: docs/sources/datasources/loki.md [hunk 2]
```diff
@@ -20,13 +20,13 @@ You can run Loki on your own hardware or use [Grafana Cloud](https://grafana.com
...
-| `Name`                | The data source name. This is how you refer to the data source in panels, queries, and Explore.                                                           |
-| `Default`             | Default data source that is pre-selected for new panels.                                                                                                  |
-| `URL`                 | URL of the Loki instance, e.g., `http://localhost:3100`.                                                                                                  |
-| `Whitelisted Cookies` | Grafana Proxy deletes forwarded cookies by default. Specify cookies by name that should be forwarded to the data source.                                  |
-| `Maximum lines`       | Upper limit for the number of log lines returned by Loki (default is 1000). Lower this limit if your browser is sluggish when displaying logs in Explore. |
+| Name               | Description                                                                                                                                               |
```
Comment (osg-grafana) [inline docs/sources/datasources/loki.md:23]:
```suggestion
| Names               | Descriptions                                                                                                                                               |
```

File: docs/sources/datasources/loki.md [hunk 3]
```diff
@@ -20,13 +20,13 @@ You can run Loki on your own hardware or use [Grafana Cloud](https://grafana.com
...
+| ------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `Name`             | The data source name. This is how you refer to the data source in panels, queries, and Explore.                                                           |
+| `Default`          | Default data source that is pre-selected for new panels.                                                                                                  |
+| `URL`              | URL of the Loki instance, e.g., `http://localhost:3100`.                                                                                                  |
+| `Accepted Cookies` | Grafana Proxy deletes forwarded cookies by default. Specify cookies by name that should be forwarded to the data source.                                  |
```
Comment (osg-grafana) [inline docs/sources/datasources/loki.md:28]:
```suggestion
| `Allowed cookies` | Grafana Proxy deletes forwarded cookies by default. Specify cookies by name that should be forwarded to the data source.                                  |
```

File: docs/sources/datasources/opentsdb.md [hunk 4]
```diff
@@ -14,15 +14,15 @@ Grafana ships with advanced support for OpenTSDB. This topic explains options, v
...
-| `Whitelisted Cookies` | List the names of cookies to forward to the data source.                                |
-| `Version`             | Version = opentsdb version, either <=2.1 or 2.2                                         |
-| `Resolution`          | Metrics from opentsdb may have datapoints with either second or millisecond resolution. |
-| `Lookup Limit`        | Default is 1000.                                                                        |
+| Name               | Description                                                                             |
+| ------------------ | --------------------------------------------------------------------------------------- |
+| `Name`             | The data source name. This is how you refer to the data source in panels and queries.   |
+| `Default`          | Default data source means that it will be pre-selected for new panels.                  |
+| `URL`              | The HTTP protocol, IP, and port of your OpenTSDB server (default port is usually 4242)  |
+| `Accepted Cookies` | List the names of cookies to forward to the data source.                                |
```
Comment (osg-grafana) [inline docs/sources/datasources/opentsdb.md:22]:
```suggestion
| `Allowed cookies` | List the names of cookies to forward to the data source.                                |
```

File: docs/sources/datasources/opentsdb.md [hunk 5]
```diff
@@ -14,15 +14,15 @@ Grafana ships with advanced support for OpenTSDB. This topic explains options, v
...
+| `Version`          | Version = opentsdb version, either <=2.1 or 2.2                                         |
+| `Resolution`       | Metrics from opentsdb may have datapoints with either second or millisecond resolution. |
+| `Lookup Limit`     | Default is 1000.                                                                        |
```
Comment (osg-grafana) [inline docs/sources/datasources/opentsdb.md:25]:
```suggestion
| `Lookup limit`     | Default is 1000.                                                                        |
```

**Review by osg-grafana — CHANGES_REQUESTED** [review]

**Comment by osg-grafana** [comment]
See style guide at https://github.com/grafana/grafana/blob/main/contribute/style-guides/documentation-style-guide.md#allowing-and-blocking.

File: docs/sources/datasources/loki.md (hunk 2 above)
Comment (ashharrison90) [reply docs/sources/datasources/loki.md:23]: i'm not sure this should be plural?

Comment (osg-grafana) [reply docs/sources/datasources/loki.md:23]: By default, use plural headings for consistency. We on the Docs Squad get to argue about such details. :)

Comment (osg-grafana) [reply docs/sources/datasources/loki.md:23]: The disadvantage of not having consistency, is that some headers will read "Name" and others will read "Names" and depending on the team who owns the technical content across entire product lines.

Comment (osg-grafana) [reply docs/sources/datasources/loki.md:23]: An advantage of using plural is that the minute you have more than one name and description, which is an ingredient for a list in the first place, the titles are already plural.

**Review by osg-grafana — APPROVED** [review]

File: docs/sources/datasources/loki.md (hunk 2 above)
Comment (osg-grafana) [reply docs/sources/datasources/loki.md:23]: > By default, use plural headings for consistency. We on the Docs Squad get to argue about such details. :)

We have "argued" and come to consensus at https://github.com/grafana/technical-documentation/issues/76 to use the singular form. :)

**Pull request merged**